edx_oauth2_provider/tests/test_create_oauth2_client.py

```diff
@@ -96,16 +98,15 @@ def test_client_creation(self, args, options):
         self.assert_client_created(args, options)
 
     @ddt.data(
-        ((URL, REDIRECT_URI), 'too few arguments'),
+        ((URL, REDIRECT_URI), 'Error'),
         ((URL, REDIRECT_URI, CLIENT_TYPES[0], CLIENT_TYPES[1]), 'unrecognized arguments'),
     )
     @ddt.unpack
     def test_argument_cardinality(self, args, err_msg):
         """Verify that the command fails when given an incorrect number of arguments."""
         with self.assertRaises(CommandError) as exc:
             self._call_command(args, {})
-
-        self.assertIn(err_msg, exc.exception.message)
+        self.assertIn(err_msg, str(exc.exception))
```

Comment:
python 2.7
exc.exception.message (is available)
python 3.6
exc.exception.message (not available)
python2.7 and python3.6
str(exc.exception) does the same as exc.exception.message and available in both.